apps/api/src/modules/etl/event-plans/create.ts

```diff
@@ -90,7 +90,7 @@ export const etlEventPlansCreate = async (
       const eventPlanDocPatch = {
         ...restOfParams,
         hostId,
-        inviteesByUserId,
+        invitees: inviteesByUserId,
```

Comment:
Because there are a handful of times when we deal with invitees, such has sending out emails or transforming them from invitees to guests, its better to be explicit about the type even in the variable name, so that when we read this document its very clear that these invitees are represented by their user ids